src/ophys_etl/utils/traces.py

```diff
@@ -1,11 +1,28 @@
 from typing import List
-from functools import partial
 import numpy as np
 from scipy.sparse import coo_matrix
 from scipy.ndimage.filters import median_filter
 
-# Partial for simplifying repeat median filter calls
-medfilt = partial(median_filter, mode='constant')
+
+def nanmedian_filter(x, filter_length):
```

Comment:
Please use a more descriptive variable name than `x` here.

Reply:
Ah, one of the things you don't see in the review is how the other functions are written, which use the same naming convention. I added typing though.

Reply:
I understand, however the single character name would make it a bit difficult to search for the variable in the function. Mostly this is to make debugging easier.